fdbserver/ClusterController.actor.cpp

```diff
@@ -4781,6 +4861,19 @@ ACTOR Future<Void> workerHealthMonitor(ClusterControllerData* self) {
 						self->excludedDegradedServers.clear();
 						TraceEvent("DegradedServerDetectedAndSuggestRecovery").log();
 					}
+				} else if (self->shouldTriggerFailoverDueToDegradedServers()) {
+					if (SERVER_KNOBS->CC_HEALTH_TRIGGER_FAILOVER) {
+						TraceEvent("DegradedServerDetectedAndTriggerFailover").log();
+						vector<Optional<Key>> dcPriority;
+						auto remoteDcId = self->db.config.regions[0].dcId == self->clusterControllerDcId.get()
+						                      ? self->db.config.regions[1].dcId
+						                      : self->db.config.regions[0].dcId;
+						dcPriority.push_back(remoteDcId);
```

Comment:
It's worth to comment that clusterController dc now becomes the remote.

Reply:
Done.